### PR TITLE
Makefile and workflow actions

### LINKED ISSUE #3: Create Makefile and Workflow Actions
- Create a makefile for the project.
- Build a workflow and set actions for the repo.

### PR DESCRIPTION
**Pull request by balandi1**
- Makefile for building, running and testing the image builder.

### CODE REVIEW

**Review by pradykaushik — CHANGES_REQUESTED** [review]
once pull request #2 has been merged, rebase and update this PR.

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      uses: actions/checkout@v1
+
+    - name: Get Dependencies using Go Modules
+      run: make dependency
+
+    - name: Build Image Builder
+      run: make
+      
+    - name: Test Image Builder
+      run: go test ./...
```
Comment (pradykaushik) [inline .github/workflows/actions.yml]: change to `make test-verbose`?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+GOMOD=$(GOCMD) mod
+BINARY=image-builder
+
+all: build
+build:
+	$(GOBUILD) -o $(BINARY)
+
+test:
+	$(GOTEST) ./...
+
```
Comment (pradykaushik) [inline Makefile:15]: add another target for verbose testing.

```
test-verbose:
        $(GOTEST) -v ./...
```

**Review by pradykaushik — CHANGES_REQUESTED** [review]

File: .github/workflows/actions.yml (hunk 1 above)
Comment (balandi1) [reply .github/workflows/actions.yml]: okay

File: Makefile (hunk 2 above)
Comment (balandi1) [reply Makefile:15]: okay

**Review by pradykaushik — APPROVED** [review]

**Pull request merged**